.github/actions/get-new-env-vars-v1/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "get-new-env-vars-v1",
+  "version": "1.0.0",
+  "description": "A GitHub action for getting new env vars for a release",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dequelabs/axe-api-team-public.git"
+  },
+  "author": "",
+  "license": "MPL-2.0",
+  "bugs": {
+    "url": "https://github.com/dequelabs/axe-api-team-public/issues"
+  },
+  "homepage": "https://github.com/dequelabs/axe-api-team-public#readme",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "ncc build src/index.ts --license ../../../licenses.txt",
+    "test": "mocha src/*.test.ts",
+    "coverage": "nyc npm run test",
+    "fmt": "prettier --write \"src/**/*.{js,ts,tsx,json,md,css,html}\"",
+    "lint": "eslint \"src/**/*.{js,ts,tsx}\""
+  },
+  "nyc": {
```

Comment:
Should probably move this stuff to a `.npmrc` file but not a huge deal

Reply:
Ahh you mean `.nycrc` file? 

https://github.com/dequelabs/axe-api-team-public/issues/109